src/containers/upload-page/store/sagas.js

```diff
@@ -1001,7 +1001,9 @@ function* uploadTracksAsync(action) {
     )
   )
 
-  if (!user.is_creator || !user.creator_node_endpoint) {
+  // If user already has creator_node_endpoint, do not reselct replica set
+  let newEndpoint = ''
+  if (!user.creator_node_endpoint) {
```

Comment:
tracks can only be uploaded if user is not a creator right? why remove that check

Reply:
didn't we decide that we were going to keep the `is_creator` flag, but not actually use it in the code base? that was the motive behind this code change

Reply:
nvm you right